pytensor/tensor/extra_ops.py

```diff
@@ -593,12 +593,12 @@ def squeeze(x, axis=None):
         # Nothing to do
         return _x
 
-    axes = ()
-    for i in axis:
-        if i < _x.ndim and not _x.broadcastable[i]:
-            axes += (i,)
-    if axes:
-        _x = specify_broadcastable(_x, axes)
+    # if the axis is of length 1 but is not specifed as broadcastable, we first explicitly specify it as broadcastable
```

Comment:
```suggestion
    # `Dimshuffle` raises when we try to drop an axis that is not statically broadcastable.
    # We add a `specify_broadcastable` instead of raising.
```